pyproject.toml

```diff
@@ -49,7 +49,7 @@ defects = ["pymatgen-analysis-defects>=2023.8.22", "shakenbreak>=3.2.0"]
 jobflow = ["jobflow[fireworks]>=0.1.14", "jobflow-remote>=0.1.0"]
 mlp = ["matgl>=1.0.0", "chgnet>=0.3.3", "mace-torch>=0.3.3", "torch-dftd>=0.4.0"]
 mp = ["atomate2>=0.0.14"]
-newtonnet = ["newtonnet>=1.1"]
+newtonnet = ["newtonnet>=1.1", "geodesic-interpolate @ git+https://github.com/virtualzx-nad/geodesic-interpolate.git"]
```

Comment:
Hi @Andrew-S-Rosen ,
I want to have tests that run for geodesic interpolate on GitHub. Is there a way you see to do that in case we are not able to put URLs?
Thanks.

Reply:
You will need to:
1. Update https://github.com/Quantum-Accelerators/quacc/blob/main/tests/requirements-newtonnet.txt to include the `geodesic-interpolate` direct link
2. Remove the direct link from pyproject.toml since we can't have direct links in there for PyPI